Tasks/DockerV2/dockerpush.ts

```diff
@@ -136,10 +137,22 @@ async function publishToImageMetadataStore(connection: ContainerConnection, imag
     const build = "build";
     const hostType = tl.getVariable("System.HostType").toLowerCase();
     const runId = hostType === build ? parseInt(tl.getVariable("Build.BuildId")) : parseInt(tl.getVariable("Release.ReleaseId"));
-    const pipelineVersion = addPipelineData ? hostType === build ? tl.getVariable("Build.BuildNumber") : tl.getVariable("Release.ReleaseName") : '';
-    const pipelineName = addPipelineData ? tl.getVariable("System.DefinitionName") : '';
-    const pipelineId = addPipelineData ? tl.getVariable("System.DefinitionId") : '';
-    const jobName = addPipelineData ? tl.getVariable("System.PhaseDisplayName") : '';
+    const pipelineVersion = addPipelineData ? hostType === build ? tl.getVariable("Build.BuildNumber") : tl.getVariable("Release.ReleaseName") : "";
+    const pipelineName = addPipelineData ? tl.getVariable("System.DefinitionName") : "";
+    const pipelineId = addPipelineData ? tl.getVariable("System.DefinitionId") : "";
+    const jobName = addPipelineData ? tl.getVariable("System.PhaseDisplayName") : "";
+    const creator = addPipelineData ? dockerCommandUtils.getCreatorEmail() : "";
+    const logsUri = addPipelineData ? dockerCommandUtils.getPipelineLogsUrl() : "";
+    const artifactStorageSourceUri = addPipelineData ? dockerCommandUtils.getPipelineUrl() : "";
```

Comment:
is it builddefinition ulr or build url?

Reply:
artifactStorageSourceUri is a part of source provenance, hence it corresponds to the definition url, which is the source of this build/run